src/languages/trino/trino.formatter.ts

```diff
@@ -136,10 +137,14 @@ export default class TrinoFormatter extends Formatter {
       reservedDependentClauses: ['WHEN', 'ELSE'],
       reservedKeywords: keywords,
       reservedFunctionNames: functions,
-      openParens: ['(', '['],
-      closeParens: [')', ']'],
+      // https://trino.io/docs/current/sql/match-recognize.html#row-pattern-syntax
+      openParens: ['(', '[', '{', '{-'],
+      closeParens: [')', ']', '}', '-}'],
```

Comment:
Wow... Looks like Trino has some pretty complex pattern matching syntax.

I would very much appreciate some tests for these patterns.